dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -318,11 +318,11 @@ BEGIN
     EDITTEXT 14002, 110, 51, 120, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
     LTEXT "", -1, 8, 64, 223, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
     CONTROL "", 14013, "Static", SS_SUNKEN | SS_OWNERDRAW, 8, 69, 10, 10
-    LTEXT "Spațiu folosit:", -1, 25, 69, 80, 10
+    LTEXT "Spaţiu utilizat:", -1, 25, 69, 80, 10
     EDITTEXT 14003, 110, 69, 86, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
     EDITTEXT 14004, 197, 69, 33, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
     CONTROL "", 14014, "Static", SS_SUNKEN | SS_OWNERDRAW, 8, 82, 10, 10
-    LTEXT "Spațiu liber:", -1, 25, 82, 80, 10
+    LTEXT "Spaţiu liber:", -1, 25, 82, 80, 10
```

Comment:
A huge thumbs-up for that kind of changes from my side! I am not sure, whether you are fully aware of it, therefore I want to highlight that the old character that you replaced here is missing in the XPSP3 Tahoma font, and therefore with changes like you did here, you do improve the compatibility here for the modules you do that change to XPSP3 and 2k3sp2. You turn a "tofu-block" then into a proper character.
If you can spot more of those characters in any ro-RO.rc then feel free to replace them. That's definitely a goodie!